packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -145,12 +137,20 @@ export default class QuanticCaseClassification extends LightningElement {
    */
   initialize = (engine) => {
     this.engine = engine;
-    this.field = CoveoHeadlessCaseAssist.buildCaseField(engine, {
-      options: {
-        field: this.coveoFieldName,
-      },
-    });
-    this.unsubscribeField = this.field.subscribe(() => this.updateFieldState());
+    try {
+      this.field = CoveoHeadlessCaseAssist.buildCaseField(engine, {
+        options: {
+          field: this.coveoFieldName,
+        },
+      });
+      this.unsubscribeField = this.field?.subscribe(() =>
+        this.updateFieldState()
+      );
+    } catch (e) {
+      if (!this.coveoFieldName) {
```

Comment:
Why is that if in a try catch and not just before that?
This seems like a required parameter no? I would assert in the constructor and then display a warning and not render anything if the coveoFieldName is not provided.

Reply:
...and why are you suppressing all other possible errors?

Reply:
Yes it's a required parameter, I think we can't access the props in the constructor🧐 or am I mistaken?